vocabularies/UI.xml

```diff
@@ -1389,6 +1390,55 @@ The trend is
         </Member>
       </EnumType>
 
+      <Term Name="Note" Type="UI.NoteType" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The annotated entity type can be visualized in a reuse component for notes" />
+        <Annotation Term="Core.LongDescription">
+          <String>Besides this annotation the reuse component also evaluates the annotations
+          [`Common.CreatedBy`](Common.md#CreatedBy),
+          [`Common.CreatedAt`](Common.md#CreatedAt),
+          [`Common.ChangedBy`](Common.md#ChangedBy),
+          [`Common.ChangedAt`](Common.md#ChangedAt)
+          on the same entity type.</String>
+        </Annotation>
+      </Term>
+      <ComplexType Name="NoteType">
+        <Annotation Term="Common.Experimental" />
+        <Property Name="Title" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Title of the note" />
+        </Property>
+        <Property Name="Content" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Content of the note, as a string" />
+          <Annotation Term="Core.LongDescription">
+            <String>The reuse component also evaluates the [`Core.MediaType`](https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Core.V1.md#MediaType)
+            annotation on this property.</String>
+          </Annotation>
+        </Property>
+        <Property Name="Type" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Type of the note" />
+        </Property>
+        <Property Name="IsLanguageDependent" Type="Edm.Boolean" Nullable="false">
+          <Annotation Term="Core.Description" String="Whether the note is language-dependent" />
+          <Annotation Term="Core.LongDescription">
+            <String>When this is true, the reuse component also evalutes the property of the same entity type
+            that is tagged with [`Common.IsLanguageIdentifier`](Common.md#IsLanguageIdentifier)
+            in order to determine the language of the note.</String>
+          </Annotation>
+        </Property>
+        <Property Name="MaximalLength" Type="Edm.Int32" Nullable="false">
```

Comment:
Isn't this the `MaxLength` of the underlying property `Content` points to via a path expression?

Reply:
Then what purpose does `_NoteBasic/NoteBasicMaximalLength` serve?

Reply:
We should ask the author of the CDS view. To me it seems totally redundant, given that `NoteBasicContent` is of type `string` and thus has a "built-in" length.

Reply:
> Then what purpose does `_NoteBasic/NoteBasicMaximalLength` serve, @thomas-kaltenbach?

From our point of view we have two entities. Note & note type.
**Note** is the content itself, with title, language and content.
**NoteType** is the configuration of a group of notes. Like "Product Description" or "Comments".
Applications can define their own note types and assign them to a SAPObjectNodeType.

The following properties...
    <Property Name="NoteBasicLanguageDependent" Type="Edm.Boolean" Nullable="false"/>
    <Property Name="NoteBasicMaximalLength" Type="Edm.Int32" Nullable="false"/>
    <Property Name="NoteBasicMultipleNotes" Type="Edm.Boolean" Nullable="false"/>
    <Property Name="NoteBasicHideTitle" Type="Edm.Boolean" Nullable="false"/>
... are all part of our NoteType. This is needed so that in the frontend is known what is allowed or not.
So an application can decide to allow only 200 characters.

The two things are now in the UI.Note entity. The question comes up regularly. @HeikoTheissen should maybe separate it and add an additional type for it, maybe `UI.NoteType`? This would make it much clearer.

Reply:
The separation into note and note type is an implementation detail, it should not be reflected in the UI annotation. But since we already have the four properties in the annotation, we don't need the `Type` itself, thus answering https://github.com/SAP/odata-vocabularies/pull/221#discussion_r1114415917.

Reply:
We need the type because we are grouping by this value. So we need to know by which field we should do this. In addition we need the "Text" of the NoteType this is description which is displayed in the UI. 
https://github.com/SAP/odata-vocabularies/pull/221#issuecomment-1412315655